pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitStatusTest.java

```diff
@@ -0,0 +1,188 @@
+package org.broadinstitute.dsm.juniperkits;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Random;
+
+import com.google.gson.Gson;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.DbTxnBaseTest;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.model.nonpepperkit.JuniperKitRequest;
+import org.broadinstitute.dsm.model.nonpepperkit.NonPepperKitCreationService;
+import org.broadinstitute.dsm.db.dto.kit.nonPepperKit.NonPepperKitStatusDto;
+import org.broadinstitute.dsm.model.nonpepperkit.NonPepperStatusKitService;
+import org.broadinstitute.dsm.model.nonpepperkit.StatusKitResponse;
+import org.broadinstitute.dsm.util.EasyPostUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+/**
+ * Tests the NonPepperStatusKitService class
+ * To run this class use the following VM variables
+ * -ea -Dconfig.file=[path to /pepper-apis/output-build-config/testing-inmemorydb.conf]
+ *  */
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
I am not familiar with this annotation. Why was it necessary?

Reply:
I found [this post](https://stackoverflow.com/questions/10806345/runwithmockitojunitrunner-class-vs-mockitoannotations-initmocksthis) about it before using it. I am still a beginner in using Mockito but this is supposed to automatically validate framework usage